diff --git src/osgDB/FileUtils.cpp src/osgDB/FileUtils.cpp
index c0c8cec88..408afe1ba 100644
--- src/osgDB/FileUtils.cpp
+++ src/osgDB/FileUtils.cpp
@@ -119,6 +119,7 @@ typedef char TCHAR;
 #include <string.h>
 
 #include <stack>
+#include <algorithm>
 
 namespace osgDB
 {
@@ -531,7 +532,9 @@ static void appendInstallationLibraryFilePaths(osgDB::FilePathList& filepath)
 #ifdef OSG_DEFAULT_LIBRARY_PATH
 
     // Append the install prefix path to the library search path if configured
-    filepath.push_back(OSG_DEFAULT_LIBRARY_PATH);
+    std::string path = OSG_DEFAULT_LIBRARY_PATH;
+    const char empty_character = '\0';
+    filepath.push_back(path);
 #endif
 }
 
