From 54540ccae7731d77bc2834885e2c88db79c4b175 Mon Sep 17 00:00:00 2001
From: Justin Carpentier <justin.carpentier@inria.fr>
Date: Tue, 14 Apr 2020 18:05:18 +0200
Subject: [PATCH 2/3] win: fix fonttype

---
 src/osgText/CMakeLists.txt | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/src/osgText/CMakeLists.txt b/src/osgText/CMakeLists.txt
index 9d64a5843..3b6422762 100644
--- a/src/osgText/CMakeLists.txt
+++ b/src/osgText/CMakeLists.txt
@@ -52,7 +52,8 @@ SET(TARGET_LIBRARIES
 
 if(OSG_TEXT_USE_FONTCONFIG AND Fontconfig_FOUND)
     MESSAGE(STATUS "osgText will be linked with FontConfig library")
-    LIST(APPEND TARGET_EXTERNAL_LIBRARIES Fontconfig::Fontconfig)
+    LIST(APPEND TARGET_EXTERNAL_LIBRARIES Fontconfig::Fontconfig )
+    LIST(APPEND TARGET_EXTERNAL_LIBRARIES LibXml2::LibXml2 )
     ADD_DEFINITIONS(-DWITH_FONTCONFIG)
 else()
     MESSAGE(STATUS "osgText will not be linked with FontConfig library")
